### ISSUE TITLE
Свойства

### ISSUE THREAD

**Issue by Red-Cat-Fat** [shark2302/TowerDefence#1]
https://github.com/shark2302/TowerDefence/blob/93f9b930581cf17a7f3c3c90dbacdc1aa9faf501/Assets/Scripts/Configs/LevelConfig.cs#L32-L99
Для таких целей в С# есть [свойства](https://docs.microsoft.com/ru-ru/dotnet/csharp/programming-guide/classes-and-structs/properties)

**Comment by shark2302** [comment]
Ну тут чисто джавистская привычка, забыл что в шарпаю такое есть, сгенерировал свойства https://github.com/shark2302/TowerDefence/commit/483f9f2713c89cc4d1e16f7fe860f29fbce5cf29, правда нигде их не использую, просто обращаюсь к публичным переменным

**Comment by Red-Cat-Fat** [comment]
У тебя публичная переменная Money, но ты заводишь для неё публичное свойство с возможностью чтения и записи. Зачем?

**Comment by shark2302** [comment]
Да, незачем, привычка просто создавать геттеры и сеттеры, они не нужны тут

**Comment by shark2302** [comment]
Убрал https://github.com/shark2302/TowerDefence/commit/42605c447a1489fb585e5a6b9d56f2a10c0cdefc

**Comment by Red-Cat-Fat** [comment]
Вообще надо было сделать переменные приватными, а свойства публичными, чтобы инкапсулировать сами значения

**Comment by shark2302** [comment]
Я просто, когда делал, брал в пример SpawnerConfig из вашего проекта, поэтому сделал публичными, думал, что они почему-то не будут отображаться в эдиторе.

**Comment by shark2302** [comment]
https://github.com/shark2302/TowerDefence/commit/6571b4f5ff6ffe0aae272da4ba9498cdbc9c9676

**Comment by Red-Cat-Fat** [comment]
Нейминг нормальный сделай и будет нормально.
Не Money и Money1, а _money и Money
